models/user-extension.js

```diff
@@ -14,9 +14,13 @@ module.exports = {
 			index: true,
 			unique: true,
 		},
-		wishlist: {
-			type: Array,
-			default: [],
-		},
+		wishlist: [
+			{
+				_id: 'ObjectId',
```

Comment:
@praveenKDass this _id we can't index? and check if we should store it as string. because if we make it object id I suppose template/details api logic may fail. please test it once